src/components/Request/NewRequestForm.Wizard.tsx

```diff
@@ -0,0 +1,193 @@
+import { useReducer } from "react";
+import { useFormContext } from "react-hook-form";
+import { Button, Spinner } from "@fluentui/react-components";
+import HiringInfo from "./NewFormSection/NewForm.HiringInfo";
+import JOA from "./NewFormSection/NewForm.JOA";
+import JobBoard from "./NewFormSection/NewForm.JobBoard";
+import LinkedInPost from "./NewFormSection/NewForm.LinkedInPost";
+import LinkedInSearch from "./NewFormSection/NewForm.LinkedInSearch";
+import RoutingInfo from "./NewFormSection/NewForm.Routing";
+import USAJobs from "./NewFormSection/NewForm.USAJobs";
+import Done from "./NewFormSection/NewForm.Done";
+
+interface INPUTSTEPFUNCTIONS {
+  next: (_methods: string[]) => string;
+  prev: (_methods: string[]) => string;
```

Comment:
Don't prefix these with an underscore -- that's special character for saying I'm not using this variable, but I am required to put it here.  When you go to "use it" but aren't going to actually use it, is when you prefix with an underscore.
For example, below, where you just want to set next to be "HiringInfo" regardless of the methods, you can write
next: (_methods) => "HiringInfo".
```suggestion
  next: (methods: string[]) => string;
  prev: (methods: string[]) => string;
```